src/en/mobile/android/development/vocal-navigation.md

```diff
@@ -0,0 +1,463 @@
+                    mAdapter.getPageTitle(i), // titre en paramètre 1 voir le xml
+                    i + 1,                    // le numéro de l’onglet actuel, paramètre 2
+                    mAdapter.getCount());     // le nombre d’onglets automatique, paramètre 3
+            if (tab != null) {
+                tab.setContentDescription(cd_param_string);
+            }         
+//For Tablayout and BottomNavigationView the "selected" information is already vocalized. 
+//Or, for example: "selected, `title tab 1`, tab 1 of 5&nbsp;".
+//or "title tab 3", tab 3 on 5.&nbsp;". Then "&nbsp;`press twice to activate&nbsp;".
+// Setting the upper terminal automatically prevents you from ending up with a vocalization of the type :
+//"selected, `title tab 4`, tab 4 on 3.&nbsp;", where the upper bound is inconsistent.
+// For the `TabHost` it would be necessary in addition to specify the information "&nbsp;selected&nbsp;".
+        }
+    }
+</code></pre><pre>
+<code class="kotlin">
+    private fun updateTabContentDescription(position: Int) {
+        for (i in 0 until mAdapter!!.count) {
+            val tab = mTabLayout!!.getTabAt(i)
+            val cd_param_string = getString(R.string.cd_tab_title_param,
+                    mAdapter!!.getPageTitle(i), // title in parameter 1 see the xml
+                    i + 1,                      // the number of the current tab, parameter 2
+                    mAdapter!!.count)           // the automatic number of tabs, parameter 3
+            if (tab != null) {
+                tab.contentDescription = cd_param_string
+//For Tablayout and BottomNavigationView the "selected" information is already vocalized. 
+//Or, for example: "selected, `title tab 1`, tab 1 of 5&nbsp;".
+//or "title tab 3", tab 3 on 5.&nbsp;". Then "&nbsp;`press twice to activate&nbsp;".
+// Setting the upper terminal automatically prevents you from ending up with a vocalization of the type :
+//"selected, `title tab 4`, tab 4 on 3.&nbsp;", where the upper bound is inconsistent.
+// For the `TabHost` it would be necessary in addition to specify the information "&nbsp;selected&nbsp;"..
+            }
+        }
+    }
+</code></pre>
+
+**Reference <abbr>WCAG</abbr>&nbsp;:**  
+- <a lang="en" href="https://www.w3.org/TR/WCAG21/#non-text-content">1.1.1 Non-text Content</a>
+
+  
+<br/><br/>
+
+## Manage the vocalization order 
+
+**Target&nbsp;:** everyone and in particular the visually impaired.  
+**When&nbsp;:** from design, and during development.
+
+**Description&nbsp;:**  
+
+The focus order of the default screen reader depends on several parameters: the "logical" reading, from left to right and from top to bottom, and the reading of the xml (order of declaration of the elements). It is possible to redefine this reading order with two tools :
+
+- accessibilityTraversalAfter: takes an id in parameter and allows to specify to the view that it must be described by the accessibility after the view passed in parameter.
```

Comment:
s/accessibilityTraversalAfter/`accessibilityTraversalAfter`/ (backquotes)

I'd say something's missing, possibly this:
s/by the accessibility/by the Accessibility API/
(same line 212)

s/after the view passed/after the view was passed/